docs/quickstart/assessment/src/main/webapp/ItemsApi.jsp

```diff
@@ -0,0 +1,33 @@
+<%@ page
+    import="com.learnosity.quickstart.ItemsApp"
+    language="java" contentType="text/html; charset=ISO-8859-1"
+    pageEncoding="ISO-8859-1"%>
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<%!
+    private ItemsApp app;
+    public void jspInit() {
+        app = new ItemsApp();
+    }
+    public void jspDestroy() {
+        app = null;
+    }
+%>
+<html>
+    <head><link rel="stylesheet" type="text/css" href="css/style.css"></head>
+    <body>
+        <h1>Standalone Assessment Example - Items API</h1>
+
+        <!-- Items API will render the assessment app into this div. -->
+        <div id="learnosity_assess"></div>
+
+        <!-- Load the Items API library. -->
+        <script src="https://items.learnosity.com/?latest"></script>
```

Comment:
@markus-liang-lrn In here, let's change this "latest" value to "latest-lts", because I really want to emphasise that clients should be selecting an LTS version.

Reply:
@TheRealEdDawson I guess , this will apply to all sdks ?

Reply:
Yes, I think so.